services/QuillLMS/db/migrate/20200604165331_migrate_lessons_to_activity.rb

```diff
@@ -18,43 +18,47 @@ def change
                           '-LKld_Wf0lzgNnEvY5aA'
                         ]
 
-      Lesson.where(uid: lessons_to_archive).each do |lesson|
-        lesson[:data]["flag"] = "archived"
-        lesson.save!
-      end
+      # The Lesson model is no longer in the code base, so this
```

Comment:
The `if defined?(Lesson)` work to skip this? I'm guessing that was it's intention. Either way is likely fine, but we could also update the `if`